doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -580,6 +589,18 @@ To launch C application next steps should be performed.
 
 <div id="run-java" class="tab-pane fade" markdown="1" >
 
+To launch Java application next steps should be performed.
+
+1. Save application code into FirstKaaDemo.java file into `demo_app` directory.
```

Comment:
Save application code into FirstKaaDemo.java file <b>in</b> `demo_app` directory.